qubes-rpc/qvm-open-in-dvm

```diff
@@ -20,10 +20,19 @@
 #
 #
 
-if ! [ $# = 1 ] && ! [ $# = 2 ]; then
+unset view_only
+view_only=
+if [ "x${1-}" = 'x--view-only' ]; then
+    view_only=--view-only
+    shift
+fi
+if [ "x${1-}" = 'x--' ]; then shift; fi
+
+if ! [ $# = 1 ]; then
 	echo "Usage: $0 [--view-only] filename"
 	exit 1
 fi
-
-# shellcheck disable=SC2016
-exec qvm-open-in-vm '$dispvm' "$@"
+case $1 in
+/*) exec qvm-open-in-vm '@dispvm' $view_only "$1";;
+*) exec qvm-open-in-vm '@dispvm' $view_only "$1";;
```

Comment:
Why this? Both cases do exactly the same thing.

Reply:
That was meant to be `"./$1"`, will fix.